test/e2e/storage/persistent_volumes-local.go

```diff
@@ -227,9 +230,13 @@ var _ = SIGDescribe("PersistentVolumes-local [Feature:LocalPersistentVolumes] [S
 		})
 	})
 
-	LocalVolumeTypes := []LocalVolumeType{DirectoryLocalVolumeType, TmpfsLocalVolumeType}
+	LocalVolumeTypes := []LocalVolumeType{DirectoryLocalVolumeType, TmpfsLocalVolumeType, GCELocalSSDVolumeType}
 
 	Context("when two pods mount a local volume at the same time", func() {
+		BeforeEach(func() {
+			framework.SkipUnlessLocalSSDExists("scsi", "fs", config.node0)
```

Comment:
We only want to skip if the type == GCELocalSSDVolumeType.

Ditto everywhere else

Reply:
In that case if we want to keep the tests as they are they will not "skip" the test perse, just skip the part that involves localSSD. The test will still show up as "passed" when running because it just wont do the localSSD part.

Alternatively I could split out the tests so that it does localSSD seperately and actually "skips" the test so it is more clear as to what is happening when you run them.

Reply:
Hmmm I wonder if it's possible to restructure the tests so that there is an It() for each type.